ruff.toml

```diff
@@ -19,6 +19,7 @@ ignore = [
     "C90",     # <...> too complex
     "COM812",  # Conflicts with formatter
     "CPY001",  # Missing copyright notice at top of file
+    "DOC501",  # Missing exceptions in docstring
```

Comment:
Why not simply add a proper docstring to the method? Should be fairly straight forward.